### ISSUE TITLE
Boost or de-positionalize audio after holding cursor over an avatar

### ISSUE THREAD

**Issue by gfodor** [mozilla/hubs#1393]
Often you want to hear someone that is far away but is too quiet due to positional audio. We should try making a feature where if you hover over their avatar, we adjust their panner node settings in some way that makes sense. (Eg turn off positional audio or boost the gain, perhaps dependent upon how far away they are.)

**Comment by gfodor** [comment]
Also: it's important this not be too aggressive -- it should basically enable people to hear others that are within some reasonable earshot more easily, not allow users to hear others who are out of earshot.

**Comment by misslivirose** [comment]
Could there be a threshold for how long you hover to trigger this, like a few seconds? If it's too quickly changed, I could see this being accidentally triggered too aggressively in a crowd, for example, and not being pleasant for someone who has noise sensitivities.

**Comment by gfodor** [comment]
yep I agree it would need some generous grace period to avoid false
positives

**Comment by robertlong** [comment]
I think it may be simpler to understand if we make it possible turn them up with the volume controls in the pause menu. There's a cap to that volume that we can boost with a gain node.

**Comment by gfodor** [comment]
Yeah part of this is UX, i think we've long needed a way to quickly adjust
volume on a person (probably temporarily, but maybe not) that doesn't
require point-and-click dexterity. If it's easy enough (like the pause
menu) it could become 'muscle memory' which would lead to different designs
for higher-order audio controls.
